lib/riemann/tools/health.rb

```diff
@@ -294,12 +295,12 @@ def darwin_memory
   def df
     case @ostype
     when 'darwin', 'freebsd', 'openbsd'
-      `df -P -t noiso9660`
```

Comment:
Fun fact: on FreeBSD this is called `cd9660`, not `iso9660`.  Will fix this in a latter commit, for now keep the bug :smile: